docs/runbooks/DeprecatedMachineType.md

```diff
@@ -0,0 +1,76 @@
+# DeprecatedMachineType
+
+## Meaning
+This alert fires when one or more Virtual Machines (VMs)  
+are using machine types that have been marked as deprecated (no  
+longer supported).
+
+## Impact
+Running VMs on deprecated machine types can cause serious issues,  
+including:
+- Loss of certain features or degraded functionality.
+- Reduced performance or unexpected behavior.
+- Security vulnerabilities due to lack of ongoing support.
+- Failure to restart VMs if the machine type is unsupported by the  
+  current or upgraded virt-launcher version.
+
+These issues can lead to service disruptions and affect workload  
+availability during or after cluster upgrades.
+
+## Diagnosis
+The alert detects VMs using deprecated machine types through **two separate
+mechanisms**, depending on the VM state:
+
+1. **Stopped VMs**
+    - Identified using the `kubevirt_vm_info` metric.
+    - These VMs are not currently running but still retain a deprecated
+      machine type in their spec.
+    - No node label is attached since the VM is not scheduled on any node.
+    - Prometheus detects them with a `unless` operation to exclude running VMs.
+
+2. **Running VMs**
+    - Identified using the `kubevirt_vmi_info` metric.
+    - These VMs are currently running on a node. The `node` label is added
+      to indicate the physical location.
+    - Prometheus uses `label_replace` to map the guest machine type to the
+      deprecated machine type list.
+    - The `group_left` operation ensures that the match is performed even if
+      some VMs do not have a corresponding node-deprecated entry.
```

Comment:
No need to explain the Prometheus alert expression.
The diagnosis step should refer them to what they need to check.